### PR TITLE
Update cm.m3u

### PR DESCRIPTION
**Pull request by boomski**

### CODE REVIEW

File: channels/cm.m3u [hunk 1]
```diff
@@ -3,5 +3,49 @@
 https://5c4984c4d9146.streamlock.net:1937/live/ngrp:GCTV_all/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/keAJteJ.png" group-title="",SCBC
 https://d202lge9i6ts6.cloudfront.net/origin1ingest1/5b70d4ab87/chunklist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.pinimg.com/originals/cc/5d/18/cc5d1807d156c21df2b866a4bdceac89.jpg" group-title="",Vision 4
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/FekWmTu.jpg" group-title="",Vision 4
 http://cdnamd-hls-globecast.akamaized.net/live/ramdisk/vision4/hls_video/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/BuKv5Vj.png" group-title="",My TV Channel
+http://connectiktv.ddns.net:5000/mytvchannel/@mytvchannel/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/mvgRAZw.jpg" group-title="Music",Play TV
+http://connectiktv.ddns.net:5000/playtv/@playtv/chunklist.m3u8
```
Comment (freearhey) [inline channels/cm.m3u]: currently not working (tested in VLC)

File: channels/cm.m3u [hunk 2]
```diff
@@ -3,5 +3,49 @@
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/YAoCSLT.jpg" group-title="",Plex TV
+http://connectiktv.ddns.net:5000/plextv/@plextv_360p+SD/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/ZTUx6q3.png" group-title="Music",Afrobit
+http://connectiktv.ddns.net:5000/afrobithd/afrobithd/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/bN6JLSn.jpg" group-title="Music",Vibe Music
+http://connectiktv.ddns.net:5000/vibetv/vibetv_360p+SD/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/3B96M7n.jpg" group-title="",Regard D'Afrik TV
+http://connectiktv.ddns.net:5000/regardafrique/regardafrique/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/QoJNAVm.jpg" group-title="",Emergence TV
+http://connectiktv.ddns.net:5000/emergencetv/emergencetv/chunklist.m3u8
```
Comment (freearhey) [inline channels/cm.m3u]: not working

File: channels/cm.m3u [hunk 3]
```diff
@@ -3,5 +3,49 @@
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/B1c3gkj.png" group-title="",Twenty Four 7 TV
+http://connectiktv.ddns.me:5000/247tv/247tv/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/1zDAaUU.png" group-title="",Dream Channel
+http://connectiktv.ddns.net:5000/dreamchannel/dreamchannel/chunklist.m3u8
```
Comment (freearhey) [inline channels/cm.m3u]: not working

File: channels/cm.m3u [hunk 4]
```diff
@@ -3,5 +3,49 @@
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/Qv1EX5K.jpg" group-title="Religious",Canal Vie
+http://connectiktv.ddns.net:5000/canalvie/canalvie/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/rQqnjof.png" group-title="Religious",Paradis TV
+http://connectiktv.ddns.net:5000/paradistv/paradistv/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/fNo0rKT.png" group-title="Religious",Repentance TV
+http://connectiktv.ddns.net:5000/repantence/repantence/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/JJMSGQz.png" group-title="",Be Loved Channel
+http://connectiktv.ddns.net:5000/belovedchannel/belovedchannel/chunklist.m3u8
```
Comment (freearhey) [inline channels/cm.m3u]: not working

File: channels/cm.m3u [hunk 5]
```diff
@@ -3,5 +3,49 @@
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/JDYKQ2L.png" group-title="Music",Hip Hop TV
+http://41.223.30.250:5000/hiphoptv/hiphoptv/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/JDYKQ2L.png" group-title="Music",Hip Hop TV
+http://connectiktv.ddns.net:5000/hiphoptv/hiphoptv/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/obD86IK.jpg" group-title="",Bikutsi TV
+http://connectiktv.ddns.net:5000/bikutsitv/bikutsi_360p+SD/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/FXHvEvJ.png" group-title="",Exel Discovery
+http://connectiktv.ddns.net:5000/exeltv/exeltv/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/Ec931Yf.jpg" group-title="",Mango TV
+http://connectiktv.ddns.net:5000/mangotv/mangotv/chunklist.m3u8
```
Comment (freearhey) [inline channels/cm.m3u]: not working

File: channels/cm.m3u [hunk 6]
```diff
@@ -3,5 +3,49 @@
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/kJncktb.jpg" group-title="",RCG TV
+http://connectiktv.ddns.net:5000/rcgtv/rcgtv/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/BMB9xq3.png" group-title="",Conso TV Afrique
+http://connectiktv.ddns.net:5000/consotv/consotv/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/RkHnwyc.jpg" group-title="",Other Channel
+http://connectiktv.ddns.net:5000/otherchannel/otherchannel/chunklist.m3u8
```
Comment (freearhey) [inline channels/cm.m3u]: not working

**Review by freearhey — CHANGES_REQUESTED** [review]
Tested in a VLC player

**Comment by boomski** [comment]
@freearhey  Very strange , I tested it also in VLC all of the links play for me , and as far as i know these channels are **not geoblocked** , Could you have a look on the website and see if you can play the streams in your browser. If they play there have a look of which version of VLC you use i tested it with version 3.0.8
http://connectik.tv/

**Comment by freearhey** [comment]
@boomski and now they work) perhaps these were only temporary failures on their server. 

As for the VLC, I also have v3.0.8 (the latest as I know)

**Review by freearhey — APPROVED** [review]

**Pull request merged**